### ISSUE TITLE
Feature Requests

### ISSUE THREAD

**Issue by Dejv311** [mpc-qt/mpc-qt#21]
Hi,
just discovered mpc-qt and it's pretty awesome! Some features I am missing:

The popup messages currently show only when increasing/decreasing playback speed and they are huge
The buttons under the seekbar had little gray dividers in the original which made it much easier to "read"
For some reason up/down is not mapped to volume up/down and I couldn't even find the option to map it so
Drag&Drop does not work for me on Ubuntu 20.04, which is kinda a bummer :/
Ctrl+A (aka always on top mode) does not switch back to "default" when pressed again like in the original

That's it for now :) Many thanks for making this marvel!

**Comment by AlexFolland** [comment]
Disclaimer: I am another user, not a developer of this project.  I am posting this commentary to help clarify this issue ticket.

> Some features I am missing:

This looks like a list of issues when comparing to some other program, not missing features.  What program?

> The popup messages currently show only when increasing/decreasing playback speed and they are huge

OSD messages appear when changing volume here and look good; can't reproduce.

![image](https://user-images.githubusercontent.com/601517/154638314-91268a3a-690e-4e0e-9405-2c6892022ccf.png)

> The buttons under the seekbar had little gray dividers in the original which made it much easier to "read"

What's "the original"?  Can you please paste a screenshot to show what you're talking about?

> For some reason up/down is not mapped to volume up/down and I couldn't even find the option to map it so

These binds are in the "Keys" menu, but labelled "Up" and "Down".  IMO they should be named "Volume Up" and "Volume Down" instead, for clarity, which would resolve this issue.  Here's a screenshot of them.  You can configure them here.

![image](https://user-images.githubusercontent.com/601517/154638582-dd6cfded-c13e-476f-9235-e036c6aebcda.png)

> Drag&Drop does not work for me on Ubuntu 20.04, which is kinda a bummer :/

What are you dragging and dropping, and from where?  I can drag subtitles onto the video and they are immediately loaded here on Manjaro XFCE with the `mpc-hc-git` AUR package.  I can also drag entries into the playlist and drag videos directly onto the viewport, which overwrites the playlist.  This is with the latest `xfwm4-git` AUR build as of this comment and I'm dragging files from the Nemo file manager.  Are you using Wayland?

> Ctrl+A (aka always on top mode) does not switch back to "default" when pressed again like in the original

What's "the original"?  My window manager lets me set any window to be always on top and bind keys to toggle it for any window.  I've just tested this Ctrl+A default bind from within mpc-qt, and it doesn't seem to work at all.  My window manager ignores mpc-qt's attempt to set the window to always-on-top mode.  When I use it while the window is not set to always-on-top mode in my window manager, the setting inside mpc-qt changes to say it's set to always-on-top mode, but the window manager's window context menu says otherwise, disagreeing.  That's weird.  I have tended to rely on my window manager for window stacking.  Making this Ctrl+A bind work in the first place seems like an important step to getting the toggling issue you've reported here resolved.

**Comment by poudink** [comment]
"The original" is MPC-HC, the media player for Windows that this is trying to replicate.

**Comment by AlexFolland** [comment]
Maybe, but MPC-HC is different in so many ways from mpc-qt that a difference must not be considered an issue.

**Comment by cmdrkotori** [comment]
^ I aim for compatible where I can, but some things such as volume being mapped to 9 and 0 like mpv instead of mpc-hc are out of necessity, due to the up and down keys navigating the playlist and mpv's feature set being different. In other areas I tried to be better than mpc-hc, for example I tried really hard to make the playlist system as robust as what you would find in clementine or xmms since mpc-hc's playlist offering is rather dumb. Other things such as ricing or the encoding support like virtualdub or webm-for-gits when I get to it are a novel addition that mpc-hc might not be capable of. So It's not an exact clone but it is close enough to be familiar to the casual user. Before we had a settings dialog mpc-qt acted almost exactly like mpc-hc's defaults.

Ctrl-A not cycling between always-on-top options is an oversight. Drag-and-drop works for me, but it does act differently if you drag it over the playlist or not. The OSD can be switched off but I suppose its size and font could be implemented easily enough. (it currently exists in the Settings dialog under the Tweaks section.)
